src/ui/js/Game.js

```diff
@@ -1917,20 +1918,39 @@ Game.pageAddSkillListFooter = function() {
       skillDesc += interactDescArray.join('\n');
     }
 
-    skillSpan = $('<span>').append($('<span>', {
+    skillSpan = $('<span>', {
       'text': skill,
-      'title': skillDesc,
       'class': 'skill_desc',
-    })).append($('<span>', {
-      'text': 'i',
-      'title': skillDesc,
-      'class': 'info_icon',
-    }));
+    });
+
+    tooltipDiv = $('<div>', {
+      'text': skillDesc,
+      'class': 'hidden_tooltip',
+    });
+
+    skillSpan.on({
+      'click': function() {
+          if ($(this).data('ui-tooltip')) {
+            $(this).tooltip('destroy');
+          } else {
+            $(this).tooltip({
+              items: '.skill_desc',
+              content: $(this).next('.hidden_tooltip').html(),
+            });
+            $(this).tooltip('open');
+          }
+        },
+      'mouseout': function(event) {
+        event.stopImmediatePropagation();
+      }
+    });
 
     if (info.code) {
       dieSkillSpanArray.push(skillSpan);
+      dieSkillSpanArray.push(tooltipDiv);
```

Comment:
Yeah, indeed, here's what the HTML looks like:

![span_div_span_div](https://cloud.githubusercontent.com/assets/2898207/16730841/1d5fd162-4743-11e6-828d-b331800c7b89.png)

That seems untidy --- is there a reason we can't encapsulate each skill's label and its help text into one HTML object of some sort?